model/converter/thrift/jaeger/test_utils.go

```diff
@@ -0,0 +1,32 @@
+package jaeger
```

Comment:
`test_utils` file gets compiled into prod. You don't need to move the functions, they will be accessible from another _test file in the same pkg.